src/TerritorialCouncil/Handlers/TerritorialCouncilCityCouncilorHandler.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\TerritorialCouncil\Handlers;
+
+use App\Entity\ElectedRepresentative\MandateTypeEnum;
+use App\Entity\TerritorialCouncil\TerritorialCouncilQualityEnum;
+
+class TerritorialCouncilCityCouncilorHandler extends AbstractTerritorialCouncilElectedRepresentativeHandler
+{
+    public function getPriority(): int
+    {
+        return TerritorialCouncilQualityEnum::QUALITY_PRIORITIES[TerritorialCouncilQualityEnum::CITY_COUNCILOR];
```

Comment:
it can be moved into parent class with: 
```php
public function getPriority(): int
{
  return TerritorialCouncilQualityEnum::QUALITY_PRIORITIES[$this->getQualityName()];
}
``` 
no?